### PR TITLE
community: dismiss stale PR reviews

### PR DESCRIPTION
**Pull request by howardjohn**
This is a trial run

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -273,6 +273,12 @@ branch-protection:
           branches: *blocked_branches
         community:
           branches: *blocked_branches
+          required_pull_request_reviews:
+            dismiss_stale_reviews: false
```
Comment (ericvn) [inline prow/config.yaml]: Just confirming as I thought you mentioned disabling reviews on a code change.  Notes say: `Set to true if you want to automatically dismiss approving reviews when someone pushes a new commit.`

Comment (ericvn) [reply prow/config.yaml]: Behavior I see implies the default is false, so this would not change that behavior at all.

Comment (howardjohn) [reply prow/config.yaml]: thanks, meant to be true

**Review by ericvn — APPROVED** [review]

**Comment by howardjohn** [comment]
ping @istio/operations-maintainers

**Review by chaodaiG — APPROVED** [review]

**Pull request merged**